clippy_lints/src/methods/filter_map.rs

```diff
@@ -171,3 +179,16 @@ pub(super) fn check<'tcx>(
             }
     }
 }
+
+fn acceptable_methods(method: &PathSegment<'_>) -> bool {
+    let methods: [Symbol; 6] = [
+        sym::clone,
+        sym::as_ref,
+        sym!(as_deref),
+        sym!(as_mut),
+        sym!(as_deref_mut),
+        sym!(to_owned),
```

Comment:
`cloned` and `copied` should also be applicable

Reply:
Sorry, I missed.

I fixed by [this commit](https://github.com/rust-lang/rust-clippy/pull/8930/commits/42cf98553a72f4b6cdac2c4c6378121d30f3bf5f), thank you!